acquia/acquia_cms
# ACMS-000: Updating year in master templates.

**Motivation**
Copyright years are incorrect.
Fixes ACMS-715

**Proposed changes**
Updates contents of footer in both master templates.

**Alternatives considered**
Updated default value of footer template, but was being overridden in master templates in which they were included.

**Testing steps**
* Navigate to any page.
* Ensure that the copyright message in the footer is "Copyright © 2021 Acquia, Inc. All Rights Reserved."

**Merge requirements**
- [x] _Bug_
